setup/src/load2/java/org/openremote/setup/load2/KeycloakSetup.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2022, OpenRemote Inc.
```

Comment:
Copyright year, as this is a new file it should be the current year. Raises a question of whether we should keep these updated when files are modified or not